afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AppStorage.java

```diff
@@ -32,6 +32,10 @@
 
     boolean isRemote();
 
+    default boolean isLocalSystem() {
```

Comment:
What is the purpose of adding this? 
It seems to be a bad design as is add implementation info in interface.